### DISCUSSION TITLE
v2.78.0

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1771] [Announcements]
## What's Changed
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1727
* 👷 Bump peter-evans/create-pull-request from 6 to 7 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1726
* 📌 Bump @tailwindcss/forms from 0.5.7 to 0.5.8 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1725
* 📌 Bump chart.js from 4.4.3 to 4.4.4 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1724
* 📌 Bump puppeteer from 23.2.0 to 23.2.2 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1723
* 📌 Bump @headlessui/react from 2.1.2 to 2.1.4 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1722
* 📌 Bump mermaid from 10.9.1 to 11.1.0 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1721
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1733
* Update users.json by @HeTongRe4per in https://github.com/nunocoracao/blowfish/pull/1730
* 📌 Bump puppeteer from 23.2.2 to 23.4.0 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1753
* 📌 Bump mermaid from 11.1.0 to 11.2.0 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1737
* 📌 Bump @tailwindcss/typography from 0.5.14 to 0.5.15 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1738
* 📌 Bump @tailwindcss/forms from 0.5.8 to 0.5.9 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1740
* 📌 Bump prettier-plugin-tailwindcss from 0.6.6 to 0.6.8 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1765
* 📌 Bump mermaid from 11.2.0 to 11.2.1 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1762
* 📌 Bump typeit from 8.8.4 to 8.8.5 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1764
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1756
* 📌 Bump puppeteer from 23.4.0 to 23.5.0 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1768
* 📌 Bump @headlessui/react from 2.1.4 to 2.1.8 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1745
* 🌐 Add Esperanto translation & update docs accordingly by @das-g in https://github.com/nunocoracao/blowfish/pull/1759
* 🖌️ Add a title for sharing via WhatsApp and Telegram. by @wermos in https://github.com/nunocoracao/blowfish/pull/1767
* Update users.json by @utrodus in https://github.com/nunocoracao/blowfish/pull/1741
* Update users.json by @LuizGlomyer in https://github.com/nunocoracao/blowfish/pull/1742
* 🧰 paginate: fix deprecation warning by @deining in https://github.com/nunocoracao/blowfish/pull/1747
* Update users.json by @Xeonzilla in https://github.com/nunocoracao/blowfish/pull/1748
* Update users.json by @RolandMarchand in https://github.com/nunocoracao/blowfish/pull/1754
* Bump micromatch from 4.0.5 to 4.0.8 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1769
* Bump dompurify from 3.0.9 to 3.1.7 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1770
* 🔖 v2.78.0 by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1720

## New Contributors
* @HeTongRe4per made their first contribution in https://github.com/nunocoracao/blowfish/pull/1730
* @das-g made their first contribution in https://github.com/nunocoracao/blowfish/pull/1759
* @utrodus made their first contribution in https://github.com/nunocoracao/blowfish/pull/1741
* @LuizGlomyer made their first contribution in https://github.com/nunocoracao/blowfish/pull/1742
* @Xeonzilla made their first contribution in https://github.com/nunocoracao/blowfish/pull/1748
* @RolandMarchand made their first contribution in https://github.com/nunocoracao/blowfish/pull/1754

**Full Changelog**: https://github.com/nunocoracao/blowfish/compare/v2.77.1...v2.78.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nunocoracao/blowfish/releases/tag/v2.78.0'>v2.78.0</a>.</em>

**Comment by designproworks** [comment]
Hi Nuno I just want to say thank you, and I appreciate your hard work. can you please add me to the authors list. 

https://designproworks.com/
